api/PayrollProcessor.Web.Api/Features/Employees/EmployeeCreate.cs

```diff
@@ -34,7 +35,7 @@ public EmployeeCreate(ICommandDispatcher dispatcher, IEntityIdGenerator generato
         OperationId = "Employees.Create",
         Tags = new[] { "Employees" })
     ]
-    public override Task<ActionResult<Employee>> HandleAsync([FromBody] EmployeeCreateRequest request, CancellationToken token)
+    public override ActionResult<Employee> Handle([FromBody] EmployeeCreateRequest request)
```

Comment:
Do you drop the `Async` suffix most of the time now? I think Microsoft s current guidance is to only use it if there is also an existing sync API of the same name...

But I've seen a lot of modern code go both ways.